app.js

```diff
@@ -23,6 +28,32 @@ app.use(function (req, res, next) {
     // Pass to next layer of middleware
     next();
 });
+//TODO create the client using the actual host when connected to dev or prod, do this by adding the host and port into the createclient() function
+var client = redis.createClient();//CREATE REDIS CLIENT
+
+
+
+//get and parse cookie and place it into req.cookies
+app.use(cookieParser());
+app.use(function(req,res,next)
+{
+
+    console.log(req.cookies.auth);
+    client.exists(req.cookies.auth, function(err, reply) {
+        if(reply == 1) {
+            console.log("Successfully Authenticated!");
+            next();
+        }
+        else{
+            //TODO go to login page maybe?
```

Comment:
Good idea, our navigation object on the frontend will soon be how we access api routes, so that would be a good place to handle this